docs/src/main/asciidoc/security-openid-connect-multitenancy.adoc

```diff
@@ -844,6 +844,81 @@ The default tenant configuration is automatically disabled when `quarkus.oidc.au
 
 Note that tenant specific configurations can also be disabled, for example: `quarkus.oidc.tenant-a.tenant-enabled=false`.
 
+== Authentication outside of the HTTP request context
+
+Sometimes it is practical to be able to create `SecurityIdentity` for given token when there is no active request context.
+The `quarkus-oidc` extension provides `io.quarkus.oidc.TenantIdentityProvider` to convert a token to a `SecurityIdentity` instance.
+When the provider is used during an HTTP request, the tenant configuration can be resolved as described in the <<static-tenant-resolution>> section.
```

Comment:
I'd like to propose adding the advice from lines 851, 852, as a NOTE when we show an example, IMHO this feature is really not about multi-tenancy, I believe the use case we discussed earlier, with only the default tenant would be typical, but indeed, we'd explain, that if needed, users can select a given tenant configuration the way you describe in the lines 850, 851. 
Adding a note that it is currently not supported for the dynamic tenant resolution is worth it too IMHO

Reply:
> Adding a note that it is currently not supported for the dynamic tenant resolution is worth it too IMHO

There is a warning bit later in the section.